### DISCUSSION TITLE
How to use Custom SSL

### DISCUSSION THREAD

**Discussion by bogordesaincom** [serversideup/docker-php#83] [Q&A]
How to using custom ssl. i was insert in dockerfile 

```dockerfile
RUN rm -rf /etc/ssl/web
RUN mkdir /etc/ssl/web
// this ssl file
COPY ./docker/ssl/ /etc/ssl/web
```

and this nginx-default file

```nginx
	# additional config
	include /etc/nginx/server-opts.d/*.conf;

    ssl_certificate /etc/ssl/web/ssl.crt;
    ssl_certificate_key /etc/ssl/web/ssl.key;
```

**Comment by jaydrogers** [comment]
I have a "beta" image coming with a lot of improvements (see the PR here: https://github.com/serversideup/docker-php/pull/59)

Here's more information on how to do it: https://github.com/serversideup/docker-php/tree/dev#production-ssl-configurations

I expect "beta" to be released to stable within the next week or two.

Hope this helps!
